cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
+
+[Term]
+id: QC:4000173
+name: MS1 signal fall (10x) count
+def: "The count of MS1 signal decline by a factor of ten or more (10x) between two subsequent scans" [PSI:QC]
+comment: An unusual high count of signal jumps or falls can indicate ESI stability issues.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-1B"  RELATED []
+
+[Term]
+id: QC:4000174
+name: Charged peptides ratio 1+ over 2+
+def: "Ratio of 1+ peptide count  over 2+ peptide count  in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A" NARROW []
+
+[Term]
+id: QC:4000175
+name: Charged peptides ratio 3+ over 2+
+def: "Ratio of 3+ peptide count over 2+ peptide count in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B" NARROW []
+
+[Term]
+id: QC:4000176
+name: Charged peptides ratio 4+ over 2+
+def: "Ratio of 4+ peptide count over 2+ peptide count in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C" NARROW []
+
+[Term]
+id: QC:4000177
+name: Mean charge in identified spectra
+def: "Mean charge in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000178
+name: Median charge in identified spectra
+def: "Median charge in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000179
+name: Charged spectra ratio +1 over +2
+def: "Ratio of 1+ spectra count over 2+ spectra count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A"  RELATED []
+
+[Term]
+id: QC:4000180
+name: Charged spectra ratio +3 over +2
+def: "Ratio of 3+ peptide count over 2+ peptide count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B"  RELATED []
+
+[Term]
+id: QC:4000181
+name: Charged spectra ratio +4 over +2
+def: "Ratio of 4+ peptide count over 2+ peptide count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C"  RELATED []
+
+[Term]
+id: QC:4000182
+name: Mean precursor charge in all MS2
+def: "Mean precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000183
+name: Median precursor charge in all MS2
+def: "Median precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000184
+name: Number of different proteins from all PSM
+def: "Number of different protein from all PSM" [PSI:QC]
```

Comment:
mhh.... does this include indistinguishable proteins, e.g. from a group where you don't know which one is present?
or only 'majority proteins' (as MQ would call them)?

Reply:
I changed it (to distinct) and added one term for indistinguishable proteins

Reply:
`Minimal set of proteins, which explains all available evidence (PSMs)`